all/HOPE-6--Per-Interface-Defaults.md

```diff
@@ -18,14 +18,16 @@ This HOPE details the addition of per-interface default parameter support to Hug
 
 ## Implementation
 
-Each interface decorator would add a new, optional, defaults argument. This argument would take a dictionary of
-parameter names and their intended defaults per the interface.
+Each interface decorator would allow addition of arbitrary kwargs to pass as defaults to the api function, should
+the api function return None.
+
+JMT TODO: Is the description above accurate? Or maybe None is allowed, but interface defaults always take precedence over functional defaults.
```

Comment:
Question for @timothycrosley

Reply:
If I understand the question correctly: I think that it's the latter, an api function can have it's own defaults but the interface defaults will take precedence if called from the specific interface.